test/e2e/removed_test.go

```diff
@@ -3,21 +3,28 @@ package e2e
 import (
 	"testing"
 
-	"github.com/openshift/console-operator/pkg/testframework"
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/console-operator/test/e2e/framework"
 )
 
+func setupRemovedTestCase(t *testing.T) (*framework.ClientSet, *operatorsv1.Console) {
+	return framework.StandardSetup(t)
+}
+
+func cleanupRemovedTestCase(t *testing.T, client *framework.ClientSet) {
+	framework.StandardCleanup(t, client)
+}
+
 // TestRemoved() sets ManagementState:Removed and verifies that all
 // console resources are deleted.
 func TestRemoved(t *testing.T) {
-	client := testframework.MustNewClientset(t, nil)
-	defer testframework.MustManageConsole(t, client)
-	testframework.MustRemoveConsole(t, client)
+	client, _ := setupRemovedTestCase(t)
+	defer cleanupRemovedTestCase(t, client)
```

Comment:
Lets call the `Standard*()` func here directly.

Reply:
interim atm, see above https://github.com/openshift/console-operator/pull/248/files#r307366710